pallets/ds-maps/src/tests.rs

```diff
@@ -0,0 +1,285 @@
+use crate::mock::*;
+use crate::{Point3D, Box3D, 
+            Point2D, Rect2D, I9F23};
+use frame_support::{
+    assert_noop, assert_ok,
+};
+use sp_std::str::FromStr;
+
+type Error = super::Error<Test>;
+type Coord = I9F23;
+
+// Constants to make tests more readable
+const ADMIN_ACCOUNT_ID: u64 = 1;
+const REGISTRAR_1_ACCOUNT_ID: u64 = 2;
+const ROOT_ID: u32 = 0;
+// this value, and values in construct() was calculated
+const AREA_ID: u16 = 58;
+const DEFAULT_HEIGHT: u16 = 30;
+
+const DELTA: &str = "0.01";
+
+fn construct_box() -> Box3D<Coord> {
+    let north_west: Point3D<Coord> = Point3D::new(Coord::from_str("55.37").unwrap(),
```

Comment:
It worth defining a private helper fn to reduce noise: `fn coord<Coord>(s: &str) -> Coord { Coord::from_str(s).unwrap() }`.